components/gitpod-protocol/src/usage.ts

```diff
@@ -0,0 +1,144 @@
+/**
+ * Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License-AGPL.txt in the project root for license information.
+ */
+
+import { WorkspaceType } from "./protocol";
+
+export interface BillableSession {
+    // The id of the one paying the bill
+    attributionId: string;
+
+    // Relevant for workspace type. When prebuild, shows "prebuild"
+    userId?: string;
+    teamId?: string;
+
+    instanceId: string;
+
+    workspaceId: string;
+
+    workspaceType: WorkspaceType;
```

Comment:
This type includes a "probe" type - not sure what that's for.

Is it work typing this as `Omit<WorkspaceType, "probe">` ?

Reply:
Good point! 🙏🏽